src/helpers/types.ts

```diff
@@ -13,6 +13,20 @@ export type Commit = {
   commitLink?: string;
 };
 
+export type GitCommit = {
```

Comment:
Ideally to use one standard data model for commit and branch, don't need to have sepearte Commit and GitCommit etc.